docs/articles/documentation/concepts/browsers.md

```diff
@@ -0,0 +1,301 @@
+
+> To test in Microsoft Edge Chromium and Legacy on the same machine, [install them side-by-side](https://docs.microsoft.com/en-us/DeployEdge/microsoft-edge-sysupdate-access-old-edge). Once they are installed, see the [Microsoft Edge Legacy Support](#microsoft-edge-legacy-support) section below for information on how to enable testing in older versions.
+
+For more information and examples, see:
+
+* Command line: [Local Browsers](../reference/command-line-interface.md#local-browsers)
+* API: [runner.browsers](../reference/api/runner/browsers.md)
+* Configuration file: [browsers](../reference/configuration-file.md#browsers)
+
+### Portable Browsers
+
+To use a portable browser, specify the path to the browser executable file. For more information and examples, see:
+
+* Command line: [Portable Browsers](../../command-line-interface.md#portable-browsers)
+* API: [runner.browsers](../reference/api/runner/browsers.md)
+* Configuration file: [browsers](../reference/configuration-file.md#browsers)
+
+### Browsers on Remote Devices
+
+To run tests on a remote mobile and desktop device, this device must have network access to the TestCafe server.
+
+First, you will need to create a remote browser connection.
+
+* Command line: specify the `remote` *alias* (see [Remote Browsers](../reference/command-line-interface.md#remote-browsers))
+* API: use the [createBrowserConnection](../reference/api/testcafe/createbrowserconnection.md) method
+* Configuration file: [browsers](../reference/configuration-file.md#browsers)
+
+After that, TestCafe will provide a URL to open on the remote device in the browser against which you want to test.
+As you open this URL, the browser connects to the TestCafe server and starts testing.
+
+> You cannot [take screenshots](../guides/basic-guides/interact-with-the-page.md#take-screenshot) or [resize the browser window](../guides/basic-guides/interact-with-the-page.md#resize-window) when you run tests in remote browsers.
+
+### Browsers in Cloud Testing Services
+
+TestCafe allows you to use browsers from cloud testing services. You can access them through [browser provider plugins](../extending-testcafe/browser-provider-plugin/README.md).
+
+The following plugins for cloud services are currently provided by the TestCafe team.
+
+Service                              | Plugin
+------------------------------------ | -------------------
+[Sauce Labs](https://saucelabs.com/)          | [testcafe-browser-provider-saucelabs](https://www.npmjs.com/package/testcafe-browser-provider-saucelabs)
+[BrowserStack](https://www.browserstack.com/) | [testcafe-browser-provider-browserstack](https://www.npmjs.com/package/testcafe-browser-provider-browserstack)
+
+You can search npm for plugins developed by the community. Their names begin with the `testcafe-browser-provider-` prefix: [https://www.npmjs.com/search?q=testcafe-browser-provider](https://www.npmjs.com/search?q=testcafe-browser-provider).
```

Comment:
For the first sentence, can we say:
You can search npm for community-developed plugins.